target/postfix/master.cf

```diff
@@ -19,7 +19,7 @@ submission     inet  n       -       n       -       -       smtpd
   -o smtpd_tls_security_level=encrypt
   -o smtpd_sasl_auth_enable=yes
   -o smtpd_sasl_type=dovecot
-  -o smtpd_sasl_path=private/auth
+  -o smtpd_sasl_path=/dev/shm/sasl-auth.sock
```

Comment:
Unrelated to this PR, but @polarathene any idea why we overwrite the default here _with the same value_? Does not make sense to me really..

Reply:
Probably just how it was originally contributed / configured (or from multiple contributions) that we got a default path in `main.cf` and another in `master.cf` :man_shrugging: 

The prior relative path was to be scoped to the chroot jail at `/var/spool/postfix`, but I guess the answer is just misconfiguration, or copy/paste from guides / snippets online without trying to understand it much :sweat_smile: 

Should be safe to drop I think, but we can do that in a future PR if preferred. I am short on time to double check.